tests/test_adapters.py

```diff
@@ -44,6 +44,8 @@ def get_error_msg_for_future(start_date, end_date):
         (Selic, date(2019, 11, 1), None, None, "1.0037"),
         (Selic, date(2019, 10, 1), None, None, "1.00751406"),
         (Selic, date(2019, 9, 1), None, None, "1.012350127488"),
+        (Cpiaucsl, date(2000, 1, 1), None, None, "1.526881275841701122268163024"),
+        (Cpiaucsl, date(2019, 1, 1), None, None, "1.023065384904600016622274640"),
```

Comment:
There is a non-documented standard for tests (I'm sorry about that, my fault! Any idea on how to document it? Maybe a docstring in the test function? IDK): 

1. Test with 1 parameter (original date)
2. Test with 2 parameters (original date plus custom value)
3. Test with 3 (parameters (original date, plus custom value, plus target date)

Can we keep this standard for CPIAUCSL as well? Also, can we keep these decorator lines in alphabetical order for better maintainability?